drivers/dma/Kconfig.siwx917

```diff
@@ -12,3 +12,8 @@ config DMA_SILABS_SIWX917_COMMON_INIT_PRIORITY
 	int "Common initialization priority"
 	depends on DMA_SILABS_SIWX917
 	default 42
+
+config SG_BUFFER_COUNT
```

Comment:
Since this option is strictly only for this driver, it should follow the same namespace, i.e. `DMA_SILABS_SIWX917_*`.